requirements/edx/development.txt

```diff
@@ -69,6 +73,7 @@ django-cors-headers==2.5.3  # via -c requirements/edx/../constraints.txt, -r req
 django-countries==5.5     # via -c requirements/edx/../constraints.txt, -r requirements/edx/testing.txt, edx-enterprise
 django-crum==0.7.9        # via -r requirements/edx/testing.txt, edx-django-utils, edx-enterprise, edx-proctoring, edx-rbac, edx-toggles, super-csv
 django-debug-toolbar==3.2.1  # via -r requirements/edx/development.in
+django-silk==4.1.0  # via -r requirements/edx/development.in
```

Comment:
do check django-silk version for python 3.8, and update accordingly

Reply:
the current one already is as per python 3.8